app/controllers/annual_leave_requests_controller.rb

```diff
@@ -22,6 +22,11 @@ def create
 
   def confirm; end
 
+  def approve
+    line_reports_leave_requests = AnnualLeaveRequest.where(user: current_user.line_reports)
+    @annual_leave_request = line_reports_leave_requests.find(params[:annual_leave_request_id])
```

Comment:
Yea that was the intention, I changed it because Brakeman flagged it as an un-scoped query issue. I'll try putting a redirect in to see if that passes the Brakeman checks.